mindsdb/integrations/handlers/clickhouse_handler/clickhouse_handler.py

```diff
@@ -31,17 +31,9 @@ def __init__(self, name, connection_data, **kwargs):
         self.connection_data = connection_data
         self.renderer = SqlalchemyRender(ClickHouseDialect)
         self.is_connected = False
-        self.protocol = connection_data.get('protocol', 'clickhouse')
-
-        # region added for back-compatibility with connections creatad before 11.05.2023
-        protocols_map = {
-            'native': 'clickhouse+native',
-            'http': 'clickhouse+http',
-            'https': 'clickhouse+https',
-        }
-        if self.protocol in protocols_map:
-            self.protocol = protocols_map[self.protocol]
-        # endregion
+        self.protocol = connection_data.get('protocol', 'clickhouse+native')
```

Comment:
Default value should be just 'native', otherwise `clickhouse+http` will be set in line #49

Reply:
Oh, yes. Changing it now